lisa/.gitignore

```diff
@@ -42,3 +42,4 @@ local.properties
 
 # UML diagrams
 diagrams/
+/lisa/
```

Comment:
We should remove this (maybe added again by my previous commits)

Reply:
Yes, it got added with a previous commit. Removed.